src/components/swap/SwapButtons/SwapButton.tsx

```diff
@@ -83,22 +109,23 @@ export const SwapButton = ({
         ) : priceImpactSeverity > PRICE_IMPACT_HIGH && !isExpertMode ? (
           t('priceImpactTooHigh')
         ) : (
-          <>
-            {t('swapWith')}
+          <Flex alignItems="center">
+            <Text fontSize={'15px'}>{t('swapWith')}</Text>
```

Comment:
```suggestion
            <Text fontSize="15px">{t('swapWith')}</Text>
```

Just to be cleaner :)